package.json

```diff
@@ -28,10 +28,10 @@
     "pug": "^2.0.0-beta6",
     "react": "^15.3.2",
     "react-dom": "^15.3.2",
-    "react-router": "^2.6.1"
+    "react-router": "^2.6.1",
+    "sass": "^0.5.0"
```

Comment:
This is probably a dev dependency.  Also, it doesn't look like we're using SASS?